numpy/core/tests/test_multiarray.py

```diff
@@ -4932,9 +4931,7 @@ def test_bytes_fields(self):
     @pytest.mark.skipif(sys.version_info[0] < 3, reason="Not Python 3")
     def test_multiple_field_name_unicode(self):
         def test_assign_unicode():
```

Comment:
Similarly here - there's no assignment any more

Reply:
@rth Still needs a fix.

Reply:
NVM, looks fixed.

Reply:
I did fix this is the last commit I think.